manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.yaml

```
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
```

```suggestion
ManifestVersion: 1.4.0
```
Since changes are needed anyways